packages/cadl-ranch-specs/http/client/structure/multi-client/mockapi.ts

```diff
@@ -0,0 +1,24 @@
+import { passOnSuccess, mockapi } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+Scenarios.Client_Structure_MultiClient = passOnSuccess([
+  mockapi.patch("/client/structure/multi-client/one", (req) => {
+    return { status: 204 };
+  }),
+  // mockapi.patch("/client/structure/multi-client/two", (req) => {
+  //   return { status: 204 };
+  // }),
```

Comment:
Commented two, four, six out, because we don't reference them in client.tsp. But my question is, how to make sure one, three, five have been called twice? @weidongxu-microsoft @timotheeguerin

Reply:
I don't have a good idea. Though I don't think this be a thing that must be verified in mockapi.

I assume as long as generated client is good, test uses both client calls 3 methods, that be enough (if endpoint is not right, mockapi would give error). No need to verify in mockapi to verify whether it is called once, or twice. If it get called a third time, it does not mean much...